src/types.ts

```diff
@@ -52,184 +43,28 @@ export type ConfigSeedArgs = PartialConfigSeedArgs & {
   uuid: string,
 }
 
-export type AnyConfigBuilder = ConfigSeed | EitherInstancesConfig
-export type PlayerConfigs = ObjectS<ConfigSeed> | Array<ConfigSeed>
-export type MachineConfigs = ObjectS<PlayerConfigs>
-
-export type ScenarioConfig = {
-  players: PlayerConfigs,
-  initialization?: Initialization,
+// export type PlayerConfigs = ObjectS<PlayerConfig>
+export type PlayerConfig = ConfigSeed
+export type InstallHapps = AgentHapp[]
+export type AgentHapp = DnaPath | DnaPath[]
+export type DnaPath = string
+
+// the mirror of InstallHapps
+export type InstalledHapps = InstalledAgentHapp[]
+
+// the mirror of AgentHapp, but more javascripty
+// and could eventually become a class
+export type InstalledAgentHapp = {
+  // the agent shared by all the Cell instances in `.cells`
+  agent: AgentPubKey
+  // the instantiated cells, which allow
+  // for actual zome calls
+  cells: Cell[]
 }
```

Comment:
This is a noteworthy part of the changes... 
as noted in the comments, this could grow and become a class... as it stands there's no real need for that

Reply:
app devs using the library would frequently interact with this struct